samples/widgets/sales-dashboard-widgets/RevenueByProduct/src/RevenueByProduct.jsx

```diff
@@ -1,20 +1,22 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please revert back the changes made to the license header year

Reply:
fixed in 87b573730d4c8021f11a7fc89783b583eed6edbf